### PR TITLE
Display events

### LINKED ISSUE #11: Event List Screen
Can view all the events is important 

 - When selecting the "Start" option from the menu, the application should redirect the user to a new screen.
 - On this screen, all recorded events should be displayed.
 - For each event, the following details must be shown:  
     - Title: The title of the event.
     - Description: The description of the event.
     - Date: The date of the event.
     - Location: The location of the event.
     - Cost: The cost of the event.

### PR DESCRIPTION
**Pull request by angel23v**
Display all the events once we click on "Iniciar" button

For more info see: https://github.com/BrightCoders-Institute/s5a1-administrador-de-eventos-team4/issues/11

### CODE REVIEW

**Comment by angel23v** [comment]
I pushed new changes

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          <p class="card-text"><strong>Fecha:</strong> <%= event.Fecha %></p>
+          <p class="card-text"><strong>Ubicacion:</strong> <%= event.Ubicacion %></p>
+          <p class="card-text"><strong>Costo:</strong> <%= event.Costo %></p>
+        </div>
+      </div>
+    </div>
+  <% end %>
+</div>
+</body>
+</html>
```
Comment (MauricioNC) [inline app/views/events/index.html.erb]: Add a new empty line after your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/events/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+          <a class="nav-link active" aria-current="page" href="./index">Iniciar</a>
+        </li>
+      </ul>
+      <form class="d-flex" role="search">
+        <input class="form-control me-2" type="search" placeholder="Buscar Evento" aria-label="Search">
+        <button class="btn btn-outline-success" type="submit">Buscar</button>
+      </form>
+    </div>
+  </div>
+</nav>
```
Comment (MauricioNC) [inline app/views/events/_navbar.html.erb]: Same, add a new empty line after your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/controllers/events_controller.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = Event.new(events_paramas)
+
+    if @event.save
+      redirect_to '/views/events/index'
```
Comment (MauricioNC) [inline app/controllers/events_controller.rb]: You can use the prefixes that rails gives you with resources.

See  the documentation at:
https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/controllers/events_controller.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  end
+
+  def index
+    @event = Event.all
+  end
+
+  private
+
+  def events_paramas
+    params.require(:event).permit(:Titulo, :Descripcion, :Fecha, :Ubicacion, :Costo)
```
Comment (MauricioNC) [inline app/controllers/events_controller.rb]: The fields in a table must be lowercase.
You can run a migration to rename this fields.

See the documentation at:
https://edgeguides.rubyonrails.org/active_record_migrations.html
https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-rename_column

Comment (MauricioNC) [inline app/controllers/events_controller.rb:26]: The fields in a table must be lowercase
You can run a migration to rename this fields.

See the documentation at:
https://edgeguides.rubyonrails.org/active_record_migrations.html
https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-rename_column

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/events/main.html.erb [hunk 5]
```diff
@@ -1,33 +1,16 @@
...
-        <h1>Sistema de Gestion de Eventos</h1>
-    </div>
+<%= render 'navbar' %>
+
+<div class="d-flex align-items-center justify-content-center container" style="margin-top: 400px">
+<h1>Sistema de Gestion de Eventos</h1>
+</div>
 </body>
-</html>
+</html>
```
Comment (MauricioNC) [inline app/views/events/main.html.erb]: Add new empty line at the end of your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by MauricioNC — APPROVED** [review]

**Review by juperameza — APPROVED** [review]
LGTM

**Pull request merged**